packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1758,6 +1758,19 @@ export const UnsupportedConfigTransformError = {
 	hint: 'See the devalue library for all supported types: https://github.com/rich-harris/devalue',
 } satisfies ErrorData;
 
+/**
+ * @docs
+ * @description
+ * Cannot use the module `astro:config` without enabling the experimental feature.
+ */
+export const CantUseAstroConfigModule = {
+	name: 'CantUseAstroConfigModule',
```

Comment:
```suggestion
export const CantUseAstroConfigModuleError = {
	name: 'CantUseAstroConfigModuleError',
```
To match all the other errors